app/src/main/java/org/simple/clinic/patientcontact/CallPatientView.kt

```diff
@@ -3,12 +3,19 @@ package org.simple.clinic.patientcontact
 import android.content.Context
 import android.util.AttributeSet
 import android.view.View
+import android.widget.Toast
 import androidx.constraintlayout.widget.ConstraintLayout
 import kotlinx.android.synthetic.main.patientcontact_view_callpatient.view.*
 import org.simple.clinic.R
 import org.simple.clinic.patient.Gender
 import org.simple.clinic.patient.displayLetterRes
 
+private typealias AgreedToVisitClicked = () -> Unit
+private typealias RemindToCallLaterClicked = () -> Unit
+private typealias RemoveFromOverdueListClicked = () -> Unit
+private typealias NormalCallButtonClicked = () -> Unit
+private typealias SecureCallButtonClicked = () -> Unit
```

Comment:
I do not understand the use of these lambas. Can you please explain?

Reply:
This will become more clear in the future PRs, but I'll try to explain here as well as I can.

In order to make a future migration away from Flow easier, I decided to implement the entire set of features (Call patient, Set appointment reminder, and Remove from overdue list) as sub views within the parent `ContactPatientBottomSheet` and switch them based on UI events. 

However, there will still only be one Mobius loop which controls the overall feature, (`ContactPatientUpdate`).

So I broke these out into separate views which the `ContactPatientBottomSheet` can manage.

These lambdas are so that the `ContactPatientBottomSheet` can register listeners on the views so that it can forward these events to the `ContactPatientUpdate`.